CodeReview.java

```diff
@@ -1,12 +1,15 @@
-public class Main {
+public class CodeReview {
```

Comment:
```suggestion
public class OnlinerTests {
```
Даем названиям классов логическое имя что они содержат